code/game/turfs/simulated/floor/mineral_floor.dm

```diff
@@ -116,9 +116,7 @@
 			spam_flag = 0
 
 /turf/open/floor/mineral/bananium/airless
-	oxygen = 0.01
-	nitrogen = 0.01
-	temperature = TCMB
+	initial_gas_mix = "o2=0.01;n2=0.01;TEMP=2.7"
```

Comment:
airless tiles either have no air or 0.01 moles each of nitrogen and oxygen
it's consistently one of those two; but it's arbitrary
I should probably make them all fully airless at some point